components/camera/client.go

```diff
@@ -227,23 +231,14 @@ func (c *client) Images(ctx context.Context, extra map[string]interface{}) ([]Na
 	images := make([]NamedImage, 0, len(resp.Images))
 	// keep everything lazy encoded by default, if type is unknown, attempt to decode it
 	for _, img := range resp.Images {
-		var rdkImage image.Image
-		switch img.Format {
-		case pb.Format_FORMAT_RAW_RGBA:
-			rdkImage = rimage.NewLazyEncodedImage(img.Image, utils.MimeTypeRawRGBA)
-		case pb.Format_FORMAT_RAW_DEPTH:
-			rdkImage = rimage.NewLazyEncodedImage(img.Image, utils.MimeTypeRawDepth)
-		case pb.Format_FORMAT_JPEG:
-			rdkImage = rimage.NewLazyEncodedImage(img.Image, utils.MimeTypeJPEG)
-		case pb.Format_FORMAT_PNG:
-			rdkImage = rimage.NewLazyEncodedImage(img.Image, utils.MimeTypePNG)
-		case pb.Format_FORMAT_UNSPECIFIED:
-			rdkImage, _, err = image.Decode(bytes.NewReader(img.Image))
-			if err != nil {
-				return nil, resource.ResponseMetadata{}, err
-			}
+		if img.MimeType == "" {
+			img.MimeType = utils.FormatToMimeType[img.Format]
```

Comment:
Is it possible for the mimetype lookup here to fail?

Reply:
Action: add TODO comment that this will be removed soon and add context on why we are adding it